app/src/main/java/com/sopt/now/compose/feature/home/UserProfileItem.kt

```diff
@@ -0,0 +1,38 @@
+package com.sopt.now.compose.feature.home
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import coil.compose.rememberAsyncImagePainter
+import com.sopt.now.compose.domain.entity.response.ResponseFollowerEntity
+
+@Composable
+fun UserProfileItem(user: ResponseFollowerEntity) {
+    Row(
+        modifier = Modifier
```

Comment:
최 상단의 modifier의 경우 확장성을 위해 매개변수로 받아오는 것이 좋습니다.
```suggestion
fun UserProfileItem(
    user: ResponseFollowerEntity,
    modifier: Modifier = Modifier
) {
    Row(
        modifier = modifier
```